### PR TITLE
chore(gatsby-design-tokens): add message when microbundle fails

### PR DESCRIPTION
**Pull request by wardpeet**
## Description
Improve dx for people using node < 10.16 and trying to run `yarn bootstrap` or building `gatsby-design-tokens`

image

### CODE REVIEW

File: packages/gatsby-design-tokens/package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "./dist/**/*"
   ],
   "scripts": {
-    "build": "microbundle build --sourcemap=false -f es,cjs && agadoo dist/index.esm.js",
+    "build": "(microbundle build --sourcemap=false -f es,cjs && agadoo dist/index.esm.js) || node -e \"console.log('\\nMicrobundle failed, make sure you\\'re using node version 10.16 or higher\\n')\"",
```
Comment (pieh) [inline packages/gatsby-design-tokens/package.json]: actually, we should still `exit 1` there

Node <10.16 might not be only reason this build fail, and we might end up publishing broken release

Comment (sidharthachatterjee) [reply packages/gatsby-design-tokens/package.json]: 😂 Sorry for adding the label!

Comment (wardpeet) [reply packages/gatsby-design-tokens/package.json]: oh good catch

Comment (wardpeet) [reply packages/gatsby-design-tokens/package.json]: image

**Comment by wardpeet** [comment]
It was super valuable, so no need to apologize!

**Pull request merged**